samples/philosophers/src/main.c

```diff
@@ -83,7 +83,7 @@
 /* end - control behaviour of the demo */
 /***************************************/
 
-#define STACK_SIZE 768
+#define STACK_SIZE (768 + CONFIG_TEST_EXTRA_STACKSIZE)
```

Comment:
How much stack space is required for philosphers on arm64?

Reply:
I'm currently using `CONFIG_TEST_EXTRA_STACKSIZE=4096`

Reply:
Just to be clear. I need a few bytes more, it works fine with `CONFIG_TEST_EXTRA_STACKSIZE=128`, I set it to 4K just because I'm not memory constrained.